api/src/main/java/org/pmiops/workbench/api/AuthDomainController.java

```diff
@@ -38,11 +44,17 @@
   @Override
   @AuthorityRequired({Authority.ACCESS_CONTROL_ADMIN})
   public ResponseEntity<Void> updateUserDisabledStatus(UpdateUserDisabledRequest request) {
-    DbUser user = userDao.findUserByEmail(request.getEmail());
-    Boolean previousDisabled = user.getDisabled();
-    DbUser updatedUser = userService.setDisabledStatus(user.getUserId(), request.getDisabled());
+    final DbUser targetDbUser = userDao.findUserByEmail(request.getEmail());
+    final Boolean previousDisabled = targetDbUser.getDisabled();
+    final DbUser updatedTargetUser =
+        userService.setDisabledStatus(targetDbUser.getUserId(), request.getDisabled());
+    authDomainAuditAdapter.fireSetAccountEnabled(
+        updatedTargetUser.getUserId(), !request.getDisabled(), !previousDisabled);
     userService.logAdminUserAction(
-        user.getUserId(), "updated user disabled state", previousDisabled, request.getDisabled());
+        updatedTargetUser.getUserId(),
```

Comment:
Do we know yet if we can phase out the legacy admin action log once we verify the action audit logs do all we need here?

Reply:
I'd file a ticket - does the new stuff cover everything and is there a need for a backfill?